src/Test.FunctionalTests.Sarif/TestData/Multitool/ValidateCommand/ExpectedOutputs/SARIF1004.ExpressUriBaseIdsCorrectly_Invalid.sarif

```diff
@@ -337,9 +337,482 @@
+              "file:///c:/log/out-err.txt"
+            ]
+          },
+          "locations": [
+            {
+              "physicalLocation": {
+                "artifactLocation": {
+                  "index": 0
+                },
+                "region": {
+                  "startLine": 72,
+                  "startColumn": 47
+                }
+              }
+            }
+          ]
+        },
+        {
+          "ruleId": "SARIF1004",
+          "ruleIndex": 1,
+          "level": "error",
+          "message": {
+            "id": "Error_UriBaseIdRequiresRelativeUri",
+            "arguments": [
+              "runs[0].invocations[0].toolExecutionNotifications[0].locations[0].physicalLocation.artifactLocation.uri",
+              "file:///C:/src/file.c"
+            ]
+          },
+          "locations": [
+            {
+              "physicalLocation": {
+                "artifactLocation": {
+                  "index": 0
+                },
+                "region": {
+                  "startLine": 26,
+                  "startColumn": 52
+                }
+              }
+            }
+          ]
+        },
+        {
+          "ruleId": "SARIF1004",
+          "ruleIndex": 1,
+          "level": "error",
+          "message": {
+            "id": "Error_UriBaseIdRequiresRelativeUri",
+            "arguments": [
+              "runs[0].invocations[0].toolConfigurationNotifications[0].locations[0].physicalLocation.artifactLocation.uri",
+              "file:///C:/src/file.c"
+            ]
+          },
+          "locations": [
+            {
+              "physicalLocation": {
+                "artifactLocation": {
+                  "index": 0
+                },
+                "region": {
+                  "startLine": 43,
+                  "startColumn": 52
+                }
+              }
+            }
+          ]
+        },
+        {
+          "ruleId": "SARIF1004",
+          "ruleIndex": 1,
+          "level": "error",
+          "message": {
+            "id": "Error_UriBaseIdRequiresRelativeUri",
+            "arguments": [
+              "runs[0].versionControlProvenance[0].mappedTo.uri",
+              "file:///C:/src/file.c"
+            ]
+          },
+          "locations": [
+            {
+              "physicalLocation": {
+                "artifactLocation": {
+                  "index": 0
+                },
+                "region": {
+                  "startLine": 95,
+                  "startColumn": 42
+                }
+              }
+            }
+          ]
+        },
+        {
+          "ruleId": "SARIF1004",
+          "ruleIndex": 1,
+          "level": "error",
+          "message": {
+            "id": "Error_UriBaseIdRequiresRelativeUri",
+            "arguments": [
+              "runs[0].originalUriBaseIds.BAD_URI_HAS_BASEID_BUT_ALSO_ABSOLUTE_URI.uri",
               "file:///C:/rules/"
             ]
           },
```

Comment:
with the skimmer fix, originalUriBaseId pops up in the violations now (yay!)

Our original implementation (which we are reverting to) missed this case, and it was undiscovered bug. Us going through the bad change actually uncovered this bug! so, cool :-).